client/components/like-button/button.jsx

```diff
@@ -83,8 +83,14 @@ var LikeButton = React.createClass( {
 
 		containerClasses = classnames( containerClasses );
 
+		if ( likeCount === 0 && ! this.props.showCount ) {
+			likeCount = '';
+		} else {
+			likeCount = likeCount + ' ';
```

Comment:
To back up a tick, the only reason the space is _inside_ the span is because React is doing whitespace removal between the spans. The like count and label are a phrase: `<span>14</span> <span>Likes</span>` would be the proper HTML, sans the classes. I think I have a way to move the space into the template.